src/bin/cli.rs

```diff
@@ -0,0 +1,281 @@
+use bitcoin::Network;
+use clap::{Parser, Subcommand};
+use clementine_cli::{deposit, withdrawal};
+
+#[derive(Parser)]
+#[command(name = "clementine")]
+#[command(about = "Clementine CLI - wallet-agnostic Citrea bridge CLI", long_about = None)]
```

Comment:
I think we should include a long_about, the long_about should talk about what the CLI allows the user to do as well as where things are stored and the `CLEMENTINE_DEBUG` environment variable so that ppl can debug on their own.